darts/models/forecasting/nlinear.py

```diff
@@ -151,7 +152,8 @@ def forward(
             )
 
             if self.normalize:
-                x = x + seq_last  # Note: works only when nr_params == 1
+                # make sure to only copy nr_params * number_of_targets
+                x = x + seq_last[:, :, : x.shape[-1]]
```

Comment:
This would have to be removed as now we perform inverse transformation twice, right?

Reply:
Yup sorry, fixing now!